src/main/java/frc/robot/subsystems/Led.java

```diff
@@ -0,0 +1,108 @@
+package frc.robot.subsystems;
+
+import org.littletonrobotics.junction.Logger;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.maps.subsystems.LedMap;
+
+public class Led extends SmartSubsystemBase {
+
+    private LedMap map;
+    private Led defaultColor;
+    private int R;
```

Comment:
Normally I'd say "these should be lowercase" but they don't actually seem to be used anyways?